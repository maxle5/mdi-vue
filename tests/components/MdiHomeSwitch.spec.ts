
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiHomeSwitch from "../../src/components/MdiHomeSwitch.vue";

test("MdiHomeSwitch snapshot", () => {
  const wrapper = mount(MdiHomeSwitch, {});
  expect(wrapper.html()).toMatchSnapshot();
});
