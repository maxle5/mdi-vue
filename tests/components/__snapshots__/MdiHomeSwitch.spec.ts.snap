// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiHomeSwitch snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M6.5 15V11H9.5V15H13V9H15L8 3L1 9H3V15H6.5M9 16V18H15V16L18 19L15 22V20H9V22L6 19L9 16M23 9H21V15H15V10H19L13.54 5.11L16 3L23 9Z" style="fill: currentColor;"></path>
</svg>"
`;
